README.md

```diff
@@ -0,0 +1,31 @@
+# Hire My Tribe
+Authors: Aaron Greenspan, Hector Huertas, Joseph "JP" Perry & Steve Pentler
+
+[Live on Heroku](https://hiremytribe.herokuapp.com/)
+
+## Overview
+
+Hire My Tribe was born to fuse "Tribes" of freelance developers and connect them to contractors.
+
+After interviewing for IT recruitment positions, we were astounded that up to 40% of developers paychecks are cashed in by recruitment firms. Afterwards, we developed a concept to connect freelance developers into "Tribes" to balance strengths, minimize culture conflicts, and allow independent developers to collectively work on larger projects.
+
+The Hire My Tribe web application features a genetic algorithm that takes in parameters such as ideal number & type of developers, hourly rates, language preferences, and skill ratings, and outputs the optimal combination of available developers based on this criteria.
+
+Technically, this Rails application allows distinct access privileges as either a developer, contractor, or admin. Depending on authorization, users are allowed to view, sort, edit, update, and delete developers, projects, specializations, and tags.
+
+## Screenshot
+
+![Screenshot](/app/assets/images/hire_my_tribe_screenshot.png)
+
+## Skills acquired
```

Comment:
Rather than say skills acquired, I'd say something like "Technologies and methodology".  The way it is emphasizes we're students a bit too much.